src/quantum/azext_quantum/operations/workspace.py

```diff
@@ -380,3 +385,62 @@ def clear(cmd):
     info = WorkspaceInfo(cmd)
     info.clear()
     info.save(cmd)
+
+
+def list_keys(cmd, resource_group_name=None, workspace_name=None):
+    """
+    List Azure Quantum workspace api keys.
+    """
+    client = cf_workspace(cmd.cli_ctx)
+    info = WorkspaceInfo(cmd, resource_group_name, workspace_name, None)
+    if (not info.resource_group) or (not info.name):
+        raise ResourceNotFoundError("Please run 'az quantum workspace set' first to select a default Quantum Workspace.")
+
+    keys = client.list_keys(resource_group_name=info.resource_group, workspace_name=info.name)
+    return keys
+
+
+def regenerate_keys(cmd, resource_group_name=None, workspace_name=None, key_type=None):
+    """
+    Regenerate Azure Quantum workspace api keys.
+    """
+    client = cf_workspace(cmd.cli_ctx)
+    info = WorkspaceInfo(cmd, resource_group_name, workspace_name, None)
+    if (not info.resource_group) or (not info.name):
+        raise ResourceNotFoundError("Please run 'az quantum workspace set' first to select a default Quantum Workspace.")
+
+    if (not key_type):
+        raise RequiredArgumentMissingError("Please select the api key to regenerate.")
+
+    keys = []
+    if key_type is not None:
+        for key in key_type.split(','):
+            keys.append(KeyType[key])
+
+    key_specification = APIKeys(keys=keys)
+    response = client.regenerate_keys(resource_group_name=info.resource_group, workspace_name=info.name, key_specification=key_specification)
+    return response
+
+
+def enable_keys(cmd, resource_group_name=None, workspace_name=None, enable_key=None):
+    """
+    Update the default Azure Quantum workspace.
+    """
+    client = cf_workspaces(cmd.cli_ctx)
+    info = WorkspaceInfo(cmd, resource_group_name, workspace_name, None)
+    if (not info.resource_group) or (not info.name):
+        raise ResourceNotFoundError("Please run 'az quantum workspace set' first to select a default Quantum Workspace.")
+
+    if (enable_key != "True") and (enable_key != "true") and (enable_key != "False") and (enable_key != "false"):
```

Comment:
nit: we can define accepted values as a list and use "in" operator. some thing like if (enable_key !in ["true", "True"])